uber/templates/regform.html

```diff
@@ -165,7 +165,7 @@
             {% popup_link "../static_views/givingExtra.html" "Why do this?" %}
             </label>
             <div class="col-sm-8">
-                {% if c.AFTER_SUPPORTER_DEADLINE and attendee.amount_extra >= c.SUPPORTER_LEVEL %}
+                {% if c.AFTER_SUPPORTER_DEADLINE and attendee.amount_extra >= c.SUPPORTER_LEVEL or c.AFTER_SHIRT_DEADLINE and attendee.amount_extra >= c.SHIRT_LEVEL %}
```

Comment:
Yeah, `and` has higher precedence than `or` so luckily this works as we want it to. I'd use parentheses but.. Django. >:[